changelog/NOTE.md

```diff
@@ -1,82 +1,130 @@
 
-   We've added an language option to the app provider which will in the end be passed to the app a user
-   opens so that the web ui is displayed in the users language.
+*   Enhancement #3180: Add canDeleteAllHomeSpaces permission
 
-   https://github.com/owncloud/ocis/issues/4367
-   https://github.com/cs3org/reva/pull/3156
-   https://github.com/owncloud/ocis/pull/4399
+   We added a permission to the admin role in ocis that allows deleting homespaces on user delete.
 
-*   Enhancement #3148: Add new jsoncs3 share manager
+   https://github.com/cs3org/reva/pull/3180
+   https://github.com/cs3org/reva/pull/3202
+   https://github.com/owncloud/ocis/pull/4447/files
 
-   We've added a new jsoncs3 share manager which splits the json file per storage space and caches
-   data locally.
+*   Enhancement #3203: Added "delete-all-spaces" permission
 
-   https://github.com/cs3org/reva/pull/3148
+   We introduced a new permission "delete-all-spaces", users holding this permission are
+   allowed to delete any space of any type.
+
+   https://github.com/cs3org/reva/pull/3203
+
+*   Enhancement #4322: OCS get share now also handle received shares
+
+   Requesting a specific share can now also correctly map the path to the mountpoint if the
+   requested share is a received share.
+
+   https://github.com/owncloud/ocis/issues/4322
+   https://github.com/cs3org/reva/pull/3200
+
+*   Enhancement #3185: Improve ldap authprovider's error reporting
+
+   The errorcode returned by the ldap authprovider driver is a bit more explicit now. (i.e. we
+   return a proper Invalid Credentials error now, when the LDAP Bind operation fails with that)
+
+   https://github.com/cs3org/reva/pull/3185
+
+*   Enhancement #3179: Improve tokeninfo endpoint
+
+   We added more information to the tokeninfo endpoint. `aliaslink` is a bool value indicating if
+   the permissions are 0. `id` is the full id of the file. Both are available to all users having the
+   link token. `spaceType` (indicating the space type) is only available if the user has native
+   access
+
+   https://github.com/cs3org/reva/pull/3179
+
+*   Enhancement #3171: Cs3 to jsoncs3 share manager migration
+
+   We added a Load() to the jsoncs3 and Dump() to the sc3 share manager. The shareid might need to be
+   prefixed with a storageid and space id.
+
+   https://github.com/cs3org/reva/pull/3171
+   https://github.com/cs3org/reva/pull/3195
+
+*   Enhancement #3204: Make the function flockFile private
+
+   Having that function exported is tempting people to use the func to get the name for calling the
+   lock functions. That is wrong, as this function is just a helper to generate the lock file name
+   from a given file to lock.
+
+   https://github.com/cs3org/reva/pull/3204
+
+*   Enhancement #3192: Enable space members to update shares
+
+   Enabled space members to update shares which they have not created themselves.
+
+   https://github.com/cs3org/reva/pull/3192
```

Comment:
https://app.codacy.com/gh/cs3org/reva/pullRequest?prid=10135807 is too stupid - it complains about "Warn for too many consecutive blank lines", but does not complain about the same layout in other places. This is the only silly thing that causes Codacy to put a "fail" on this PR.

Reply:
fixed it